config/settings.yml

```diff
@@ -1216,6 +1220,7 @@
       :authentication_check_interval: 1.hour
       :chargeback_generation_interval: 1.day
       :chargeback_generation_time_utc: 01:00:00
+      :compliance_purge_interval: 1.week
```

Comment:
Looking at the existing schedules for some of the other bigger tables like `event_streams`, `policy_events` and `notifications`, it looks like the purge interval is `1.day`. I'm wondering if it should be that for `compliances` too because that table has been know to get very big too. I would expect that the `policy_events` table would be updated at a similar rate as `compliances`